### DISCUSSION TITLE
MapLibre icon now available at simpleicons.org

### DISCUSSION THREAD

**Discussion by elliotwutingfeng** [maplibre/maplibre-gl-js#2715] [Show and tell]
I've [added](https://github.com/simple-icons/simple-icons/pull/7187) the MapLibre [icon](https://simpleicons.org/?q=maplibre) to <https://simpleicons.org>

<img src="https://simpleicons.org/icons/maplibre.svg"  width="48" height="48" />

Badges from <https://shields.io> should be available from the first week of July (2-4 July 2023). If you can see the logo in the badges below, it is available.

![MapLibre](https://img.shields.io/badge/Preview-396CB2?style=plastic&logo=maplibre&logoColor=ffffff&label=MapLibre)
![MapLibre](https://img.shields.io/badge/Preview-396CB2?style=flat&logo=maplibre&logoColor=ffffff&label=MapLibre)
![MapLibre](https://img.shields.io/badge/Preview-396CB2?style=flat-square&logo=maplibre&logoColor=ffffff&label=MapLibre)
![MapLibre](https://img.shields.io/badge/Preview-396CB2?style=for-the-badge&logo=maplibre&logoColor=ffffff&label=MapLibre)
![MapLibre](https://img.shields.io/badge/Preview-396CB2?style=social&logo=maplibre&logoColor=000000&label=MapLibre)

**Comment by HarelM** [comment]
Thanks!! Not sure when this is useful, but it can't hurt 🙂
